### PR TITLE
Replace ``pygraphviz`` with ``graphviz``

### PR DESCRIPTION
**Pull request by chensgit169**
Considering python package ``graphviz`` is more lightweight, installation-flexible, well-documented and simple to use, implemented draw_dag() with ``graphviz`` and removed ``pygraphviz``. Dependence upon ``IPython`` was also removed since this is not generally compatiable with different ways to run ``pyquafu``. Instead supports for notebook users should be tested and documented independently in the later development -- luckily, it seems  `graphviz`` does well in the aspect.

### CODE REVIEW

File: quafu/tasks/tasks.py [hunk 1]
```diff
@@ -253,9 +253,7 @@ def send(self,
             raise UserError()
         else:
             res_dict = response.json()
-            import pprint
```
Comment (Zhaoyilunnn) [inline quafu/tasks/tasks.py:256]: I assume that you want to use the verbose flag to control whether we print response, so you finally decide to remove this now? If so, maybe the verbose flag should be deleted.

File: quafu/visualisation/draw_dag.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        Args:
+            qc (QuantumCircuit): pyquafu quantum circuit if provided
+            dag (DAG): DAG object with nodes and edges, built from qc if not provided
+            output_format (str): output format, including "png", "svg", "pdf"...
+            output_filename (str): file name of generated image
+
+        Returns:
+            dot: graphviz.Digraph object
+
+        """
```
Comment (Zhaoyilunnn) [inline quafu/visualisation/draw_dag.py:37]: Just a minor issue, align the indent ^_^

**Review by Zhaoyilunnn — COMMENTED** [review]
Seems that `graphviz` works well, there are just two minor comments for your reference

File: quafu/tasks/tasks.py (hunk 1 above)
Comment (chensgit169) [reply quafu/tasks/tasks.py:256]: Actually I intended to help debug by setting ``verbose=True``, yet it was not completed. Let's delete it for now. By the way I just noticed status_code in two level in master branch were not seperated yet?

File: quafu/visualisation/draw_dag.py (hunk 2 above)
Comment (chensgit169) [reply quafu/visualisation/draw_dag.py:37]: Yes, the docstring needs updating as well. I will finish it later.

File: quafu/tasks/tasks.py (hunk 1 above)
Comment (Zhaoyilunnn) [reply quafu/tasks/tasks.py:256]: I believe this is a mistake I made when I merge stable/0.3 branch into master. When resolving conflicts, I mistakenly removed the change in stable/0.3 branch and kept the master branch change. I will fix this later.

**Review by Zhaoyilunnn — APPROVED** [review]

**Pull request merged**